src/View/Helper/FormHelper.php

```diff
@@ -572,6 +572,16 @@ public function secure(array $fields = [], array $secureAttributes = [])
             'value' => $tokenData['unlocked'],
         ]);
         $out .= $this->hidden('_Token.unlocked', $tokenUnlocked);
+        if (Configure::read('debug')) {
```

Comment:
It would be better to accept a param in form->create() for enabling or disabling this, defaulting to the configure debug value. That way this method will not be called for each input in the form